include/edac.h

```diff
@@ -0,0 +1,255 @@
+/*
```

Comment:
Should we put the header to some subdirectory, perhaps drivers? Your have `ZEPHYR_INCLUDE_DRIVERS_EDAC_H_` already mentioning drivers.

Reply:
moved to `include/drivers/`